table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java

```diff
@@ -196,27 +198,27 @@ private void mount(List<Table> tables, UdbBypassSpec bypassSpec)
         continue;
       }
       AlluxioURI tableUfsUri = new AlluxioURI(tableUfsPath);
-      AlluxioURI parentUfsUri = tableUfsUri.getParent();
-      colocatedTables.put(parentUfsUri, table);
+      AlluxioURI rootUfsUri = new AlluxioURI(tableUfsUri, "/", false);
+      colocatedTables.put(rootUfsUri, table);
```

Comment:
After discussion with @maobaolong, I extended the scope of mount to the root of UFS. This is based on the assumption that if the table is accessible by Alluxio, then any leading path before the table, including the root, is accessible, too. In other words, we only need to mount the roots of every UFS to be able to access the tables contained in various locations in the UFS's, instead of finding and mounting them one by one. I tested it with a non-clustered setup of Hive and Presto, and it worked.

Would appreciate your comments on this approach, @apc999 .